modules/data.land/R/download_NEON_soildata.R

```diff
@@ -0,0 +1,85 @@
+##' @name download_NEON_soildata
+##' @description: 
+##' Download NEON Soil Water Content and Soil Salinity data by date and site name
+##' 
+##' @param site four letter NEON site code name(s). If no site is specified, it will download all of them (chr) (e.g "BART" or multiple c("SRER", "KONA", "BART"))
+##' @param avg averaging interval: either 30 minute OR 1 minute, default will return BOTH (chr) (e.g 1 or 30) 
+##' @param var variable of interest, either "SWC" (soil water content) or "SIC" (soil ion content)
+##' @param startdate start date as YYYY-mm. If left empty, all data will available will be downloaded (chr)
+##' @param enddate start date as YYYY-mm. If left empty, all data will available will be downloaded (chr) 
+##' @param outdir out directory to store .CSV files with individual site data
+##' @return Data.frames for each site of interest loaded to the R Global Environment, AND saved as .csv files
+##' 
+##' @author Juliette Bateman
+##' 
+##' @example
+##' \dontrun{
+##' test <- download_NEON_soildata(
+##'   site = c("SRER", "BART"),
+##'   avg = 30,
+##'   startdate = "2019-01",
+##'   enddate = "2020-01",
+##'   var = "SWC",
+##'   outdir = getwd())}
+
+## Install NEON libs
+#devtools::install_github("NEONScience/NEON-geolocation/geoNEON")
+#devtools::install_github("NEONScience/NEON-utilities/neonUtilities", force = TRUE)
+#install.packages("BiocManager")
+# BiocManager::install("rhdf5")
+
+
+download_NEON_soildata <- function(site, avg = "all", var,
+                                   startdate = NA, enddate = NA,
+                                   outdir) {
+  
+  
+  #################### Data Download from NEON #################### 
+  soil.raw = neonUtilities::loadByProduct(dpID = "DP1.00094.001", site = "KONA", avg = avg, startdate = startdate, enddate = enddate, check.size = FALSE)
```

Comment:
Why is there a specific site being hardcoded here, rather than the sites passed in as arguments?